src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseDetailsEntity.java

```diff
@@ -79,6 +79,7 @@
     public static final String SUPPLEMENTARY_DATA_COL = "supplementary_data";
     public static final String LOCATION = "data.caseManagementLocation.baseLocation";
     public static final String REGION = "data.caseManagementLocation.region";
+    public static final String CASE_ACCESS_CATEGORY = "CaseAccessCategory";
```

Comment:
```suggestion
    public static final String CASE_ACCESS_CATEGORY = "data.CaseAccessCategory";
```
Since it's a top level field that will be in the case data.